app/code/Magento/Catalog/Block/Product/View.php

```diff
@@ -384,4 +385,24 @@ protected function getCustomerId()
     {
         return $this->customerSession->getCustomerId();
     }
+
+    /**
+     * Retrieves a product`s price info
+     * @param string $priceType
+     * @return \Magento\Framework\Pricing\Price\PriceInterface
+     */
+    public function getProductPriceInfo($priceType = Price\RegularPrice::PRICE_CODE)
+    {
+        return $this->getProduct()->getPriceInfo()->getPrice($priceType);
+    }
+
+    /**
+     * Render a product amount with the default precision
+     * @param Pricing\Amount\AmountInterface $amount
+     * @return float
+     */
+    public function renderAmount(Pricing\Amount\AmountInterface $amount)
+    {
+        return $this->priceCurrency->round($amount->getValue());
```

Comment:
I don't think such responsibility needs to be added back to the block. It was intentionally removed in https://github.com/magento/magento2/commit/1527713f82aa535692d23b3f5c432b80b13f89ae.

Looks like method like `\Magento\Framework\Pricing\Render\PriceBox::renderAmount` needs to be called instead.